lib/websockets.py

```diff
@@ -34,6 +34,7 @@
     sys.exit("install SimpleWebSocketServer")
 
 from . import util
+from lib import bitcoin
```

Comment:
use a relative import; especially as the package structure is different when running from source and when having the lib installed
( https://github.com/spesmilo/electrum/issues/4279 )

Reply:
I did not know that. Will fix.